web/src/features/auth/slices/authenticationSlice.ts

```diff
@@ -160,15 +171,17 @@ export const authenticate = (): AppThunk => dispatch => {
   }
 }
 
-export const signout = (): AppThunk => async dispatch => {
-  try {
-    const postLogoutRedirectURI = window.location.href
-    const returl = encodeURIComponent(
-      `${KC_AUTH_URL}/realms/${KC_REALM}/protocol/openid-connect/logout?post_logout_redirect_uri=${postLogoutRedirectURI}`
-    )
-    const logoutURL = `${SM_LOGOUT_URL}${returl}`
-    window.location.href = logoutURL
-  } catch (e) {
-    return dispatch(signoutError(`Failed to signout: ${e}`))
+export const signout =
+  (idToken?: string): AppThunk =>
+  async dispatch => {
+    try {
+      const postLogoutRedirectURI = window.location.href
+      const returl = encodeURIComponent(
```

Comment:
I know I'm being v picky now, but I'd prefer a different name for this variable, so it's clearer what it is.